solutions/security/ai/ai-assistant.md

```diff
@@ -58,7 +53,13 @@ While AI Assistant is compatible with many different models, refer to the [Large
 
 ::::
 
+### Elastic Managed LLM [elastic-managed-llm-security-ai-assistant]
 
+[Elastic Managed LLM](https://www.elastic.co/docs/reference/kibana/connectors-kibana/elastic-managed-llm) is the default large language model (LLM) connector available in the AI Assistant for eligible users. It provides immediate access to generative AI features without requiring any setup or external model integration.
+
+The Elastic Managed LLM is available out-of-the box, no manual connector setup or API key management is required for initial use. However, you can configure and use a third party LLM connector, such as OpenAI, Azure, or Amazon Bedrock if you prefer.
```

Comment:
nit: I think `box, no` should either be `box; no` or `box, and no`

Reply:
Thanks, Brandon. Updated via [6ebabde](https://github.com/elastic/docs-content/pull/1778/commits/6ebabde0e850cee5f7ae01795f7eb9fd8ae0d0c4).